src/views/MondoMarket/index.jsx

```
@@ -0,0 +1,270 @@
+  getMondoByCategory,
+  getMondoMarketCategories,
+  getTopMondoMarket,
+  searchMondoItem,
+} from "../../app/features/mondomarket";
+
+import { MondoDetailsViewer } from "../../services/mondomarket";
+
+const MondoMarket = () => {
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const [addModal, setAddModal] = useState(false);
```

ESLint

[error] 'addModal' is assigned a value but never used.

(no-unused-vars)